packages/nitro-protocol/contracts/test/TestEthAssetHolder2.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.4;
+pragma experimental ABIEncoderV2;
+import '../ETHAssetHolder.sol';
+
+/**
+ * @dev This contract is a clone of the ETHAssetHolder contract. It will be initialized to point to the TestNitroAdjudicator.
+ */
+contract TestEthAssetHolder is ETHAssetHolder {
+    uint256 public dummy2; // this is simply to make the contract have distinct bytecode from the ETHAssetHolder (otherwise bytecode verification can fail)
```

Comment:
The only difference between this file and `TestEthAssetHolder` seems to be `dummy` vs `dummy2`. What's the purpose of this file?

Reply:
Ok, I figured it out by reading the tests. I think there are tests that use multiple EthAssetHolders, and you wanted to use different addresses for them? It's hard to tell for sure.